docker/docker-compose.dev.yml

```diff
@@ -5,17 +5,14 @@ services:
     build:
       context: ..
       dockerfile: docker/Dockerfile
-      args:
-        env: "development"
-        dev: ""
-        simulate_removal: "-s"
+      target: dev
     image: libero/search_dev:${IMAGE_TAG:-master}
     volumes:
       - ..:/srv/app
       - ./settings.dev.py:/srv/app/search/settings.py
     command: "flask run --host=0.0.0.0"
     healthcheck:
-        test: ["CMD", "nc", "-z", "localhost", "5000"]
+        test: ["CMD-SHELL", "bash -c 'echo > /dev/tcp/localhost/5000'"]
```

Comment:
never seen `CMD-SHELL`, but if it works...

Reply:
It uses shell rather than bash. I can't get it to work with using CMD.